### PR TITLE
fix: Font size of navigation bar title in the contacts tab

### PR DESCRIPTION
**Pull request by agisilaos**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

With this PR we fix the font size of navigation bar title in the contacts tab.

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: Wire-iOS/Sources/Helpers/UINavigationItem+NavigationBarTitle.swift [hunk 1]
```diff
@@ -20,7 +20,7 @@ import UIKit
 
 extension UINavigationItem {
 
-    func setupNavigationBarTitle(title: String) {
+    func setupNavigationBarTitle(title: String?) {
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/Helpers/UINavigationItem+NavigationBarTitle.swift:23]: Why is this change needed?

Comment (agisilaos) [reply Wire-iOS/Sources/Helpers/UINavigationItem+NavigationBarTitle.swift:23]: @johnxnguyen If I tried to use a guard let statement in `StartUIViewController.swift` where we set the time then I would have problems in tests where it would generate nothing no title, no content for some reason. Not sure if that's the right approach and that was one of the things I wanted to discuss with you.

Comment (agisilaos) [reply Wire-iOS/Sources/Helpers/UINavigationItem+NavigationBarTitle.swift:23]: @johnxnguyen So If I do something like this in `StartUIViewController`, it doesn't work in tests

```
if let team = (selfUser as? ZMUser)?.team {
            guard let title = team.name?.capitalized else { return }
            navigationItem.setupNavigationBarTitle(title: title)
        } else {
            guard let title = selfUser.name?.capitalized else { return }
            navigationItem.setupNavigationBarTitle(title: title)
        }
```

Screenshot 2022-11-04 at 10 20 32

**Review by KaterinaWire — APPROVED** [review]

**Review by johnxnguyen — APPROVED** [review]
